modules/client-configuration/pages/clients-rh-rhui.adoc

```diff
@@ -0,0 +1,424 @@
+| {redhat} 6 | RHEL6-Pool for x86_64 | rhel6-pool-x86_64
+| {redhat} 7 | RHEL7-Pool for x86_64 | rhel7-pool-x86_64
+| {redhat} 8 | RHEL8-Pool for x86_64 | rhel8-pool-x86_64
+|===
+
+
+include::snippets/manual_channels.adoc[]
+
+[IMPORTANT]
+====
+For {redhat} 8 clients, add both the Base and Appstream channels.
+You will require packages from both channels.
+If you do not add both channels, you will not be able to create the bootstrap repository, due to missing packages.
+====
+
+
+include::snippets/manual_associate.adoc[]
+
+
+
+== Add Software Channels
+
+Before you register {redhat} clients to your {productname} Server, check that you have the {redhat} product enabled, and the required channels are fully synchronized.
+
+ifeval::[{suma-content} == true]
+
+Your {susemgr} subscription entitles you to the tools channels for {sleses} (also known as {redhat} Expanded Support or RES).
+You must use the client tools channel to create the bootstrap repository.
+This procedure applies to both Salt and traditional clients.
+endif::[]
+
+ifeval::[{suma-content} == true]
+
+The products you need for this procedure are:
+
+[[redhat-channels-wizard]]
+[cols="1,1,1", options="header"]
+.Red Hat Products - WebUI
+|===
+| OS Version | Base Channel                     | Pool Channel
+| {redhat} 6 | RHEL6 Base x86_64                | RHEL6-Pool for x86_64
+| {redhat} 7 | RHEL7 Base x86_64                | RHEL7-Pool for x86_64
+| {redhat} 8 | RHEL or SLES ES or CentOS 8 Base | RHEL8-Pool for x86_64
+|===
+
+
+include::snippets/add_channels_wizard.adoc[]
+
+
+endif::[]
+
+ifeval::[{uyuni-content} == true]
+
+The channels you need for this procedure are:
+
+[[redhat-channels-cli]]
+[cols="1,1,1,1", options="header"]
+.Red Hat Channels - CLI
+|===
+
+| OS Version
+| Base Channel
+| Client Channel
+| Tools Channel
+
+| {redhat} 6
+| rhel-x86_64-server-6
+| -
+| res6-suse-manager-tools-x86_64
+
+| {redhat} 7
+| rhel-x86_64-server-7
+| -
+| res7-suse-manager-tools-x86_64
+
+| {redhat} 8
+| rhel-x86_64-server-8
+| -
+| res8-suse-manager-tools-x86_64
+
+|===
+
+
+include::snippets/add_channels_cli.adoc[]
+
+
+[NOTE]
+====
+The client tools channel provided by [command]``spacewalk-common-channels`` is sourced from {uyuni} and not from {suse}.
+====
+
+endif::[]
+
+
+To use RHUI, you need to manually add the required HTTP headers to the configuration file.
+Without them, you cannot successfully perform a client synchronization.
+
+
+
+.Procedure: Adding HTTP Headers to the Configuration File
+. Locate the [systemitem]``X-RHUI-ID`` and [systemitem]``X-RHUI-SIGNATURE`` HTTP headers from your RHUI instance.
+. Open the [path]``/etc/rhn/spacewalk-repo-sync/extra_headers.conf`` configuration file, and add or edit these lines with the correct information:
++
+----
+[channel_label]
+￼X-RHUI-ID=value
+￼X-RHUI-SIGNATURE=value
+----
+. Alternatively, use these commands to locate the information and add it to the configuration file:
```

Comment:
This is correct, but might be misinterpreted as "add these commands to the configuration file", which would be wrong.

What about "Alternatively, use these commands on the RedHat instance to locate the information. Then add their result to the configuration file on the SUSE Manager Server."

Reply:
OK, will rework this procedure.